### PR TITLE
first simple readme added

### PR DESCRIPTION
**Pull request by hendrikebbers**
first simple readme added. See #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# IcedTeaWeb
+This repository contains all sources of IcedTeaWeb. 
+
+## About IcedTeaWeb
+IcedTeaWeb is an open source implementation of JSR-56 that is better know as WebStart.
```
Comment (AndreasEhret) [inline README.md]: I would add "Java" the the end of the sentence to make description compliant to Wikipedia: "...that is better know as Java Web Start."

Comment (hendrikebbers) [reply README.md]: can you do this?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Development origin
+The project was former hosted as part of the classpath.org project but moved to github.
```
Comment (AndreasEhret) [inline README.md]: "formerly" instead of "former"?

Should we add the link to the web site?: https://icedtea.classpath.org

File: README.md (hunk 1 above)
Comment (AndreasEhret) [reply README.md]: yes, will make my changes on the branch

**Pull request merged**